src/prefect/environments/storage/bitbucket.py

```diff
@@ -57,12 +57,6 @@ def __init__(
         self.path = path
         self.ref = ref
 
-        super().__init__(**kwargs)
```

Comment:
Oops looks like an accidental delete of the super 🙂

Reply:
Yup, I goofed that one.  Also need to update the unit tests to remove the "bitbucket-flow-storage" label.